### PR TITLE
added scale 44 group structure

### PR DESCRIPTION
**Pull request by shimwell**
This PR is an attempt to add SCALE-44 to the collection of energy group structures as requested on the NEA course and mentioned in issue #2043

### CODE REVIEW

**Review by nelsonag — CHANGES_REQUESTED** [review]
Hi @shimwell, I just took a look and put in a few minor comments. In general, I think the SCALE-44 structure is used widely enough to warrant being included with the structures we already have present.

Thanks!

File: openmc/mgxs/__init__.py [hunk 1]
```diff
@@ -67,7 +71,12 @@
     0.2e6, 0.3e6, 0.4e6, 0.45e6, 0.51e6, 0.512e6, 0.6e6, 0.7e6, 0.8e6, 1.e6,
     1.33e6, 1.34e6, 1.5e6, 1.66e6, 2.e6, 2.5e6, 3.e6, 3.5e6, 4.e6, 4.5e6, 5.e6,
     5.5e6, 6.e6, 6.5e6, 7.e6, 7.5e6, 8.e6, 10.e6, 12.e6, 14.e6, 20.e6, 30.e6,
-    50.e6])
+    50.e6]),
```
[the remark this answers is not in the document]
Comment (shimwell) [reply openmc/mgxs/__init__.py]: Thanks, I've removed the comma now

File: openmc/mgxs/__init__.py [hunk 2]
```diff
@@ -67,7 +71,12 @@
...
+GROUP_STRUCTURES['SCALE-44'] = np.array([3.e-3, 7.5e-3, 1.e-2, 2.53e-2, 3.e-2,
```
[the remark this answers is not in the document]
Comment (shimwell) [reply openmc/mgxs/__init__.py]: Thanks yes that should be added

File: openmc/mgxs/__init__.py [hunk 3]
```diff
@@ -67,7 +71,12 @@
...
+    4.e-2, 5e-2, 7.e-2, 1.e-1, 1.5e-1, 2.e-1, 2.25e-1, 2.5e-1, 2.75e-1,
+    3.25e-1, 3.5e-1, 3.75e-1, 4.e-1, 6.25e-1, 1., 1.77, 3., 4.75, 6., 8.1,
+    1.e1, 3.e1, 1.e2, 5.5e2, 3.e3, 1.7e4, 2.5e4, 1.e5, 4.e5, 9.e5, 1.4e6,
+    1.85e6, 2.354e6, 2.479e6, 3.e6, 4.8e6, 6.434e6, 8.1873e6, 2.e7]),
```
[the remark this answers is not in the document]
Comment (shimwell) [reply openmc/mgxs/__init__.py]: thanks

**Review by nelsonag — APPROVED** [review]

**Comment by paulromano** [comment]
@shimwell Thanks for this PR. While you're at it, can you fix the following line? Looks like an extra comma snuck into one of the dictionary keys by accident.
```diff
diff --git a/openmc/mgxs/__init__.py b/openmc/mgxs/__init__.py
index 133b4c257..938dd8c7a 100644
--- a/openmc/mgxs/__init__.py
+++ b/openmc/mgxs/__init__.py
@@ -154,7 +154,7 @@ GROUP_STRUCTURES['VITAMIN-J-175'] = np.array([
     1.1052e7, 1.1618e7, 1.2214e7, 1.2523e7, 1.2840e7, 1.3499e7,
     1.3840e7, 1.4191e7, 1.4550e7, 1.4918e7, 1.5683e7, 1.6487e7,
     1.6905e7, 1.7332e7, 1.9640e7])
-GROUP_STRUCTURES['TRIPOLI-315,'] = np.array([
+GROUP_STRUCTURES['TRIPOLI-315'] = np.array([
     1.0e-5, 1.1e-4, 3.000e-3, 5.500e-3, 1.000e-2, 1.500e-2, 2.000e-2, 3.000e-2,
     3.200e-2, 3.238e-2, 4.300e-2, 5.900e-2, 7.700e-2, 9.500e-2, 1.000e-1,
     1.150e-1, 1.340e-1, 1.600e-1, 1.890e-1, 2.200e-1, 2.480e-1, 2.825e-1,
```

**Review by paulromano — APPROVED** [review]

**Pull request merged**